makefile

```diff
@@ -232,6 +233,8 @@ $(buildDir)/dist.tar.gz:$(buildDir)/make-tarball $(clientBinaries) $(uiFiles)
 	./$< --name $@ --prefix $(name) $(foreach item,$(distContents),--item $(item)) --exclude "public/node_modules" --exclude "clients/.cache"
 # end main build
 
+update-lobster: 
```

Comment:
It seems like this target is assuming that `bin/.lobster-temp` is not there already (I believe git clone will fail otherwise). Can we make it so that there's a new target (e.g. `clean-lobster`) that's depended on by both the `update-lobster` and the `clean` targets so that it automatically deletes `bin/.lobster-temp` before either target runs?
Like:
```make
update-lobster: clean-lobster
<stuff...>
clean: clean-lobster
<more stuff...>
clean-lobster:
	rm -rf $(lobsterTempDir)
```